runtime/lua/vim/ui.lua

```diff
@@ -103,4 +103,51 @@ function M.input(opts, on_confirm)
   end
 end
 
+
+--- Prompts the user to pick an option from a list of choices
+---
+---@param msg string Text asked
+---@param opts table
+---     - choices (table|nil)
```

Comment:
EmmyLua supports "array" syntax:

```suggestion
---     - choices (string[]|nil)
```